README.rst

```diff
@@ -304,7 +304,7 @@ NI Switch Executive Python API Status
 +-------------------------------+---------------------+
 | NI Switch Executive (nise)    |                     |
 +===============================+=====================+
-| Driver Version Tested Against | 20.0.0              |
+| Driver Version Tested Against | 20.5.0              |
```

Comment:
You said:

> Update "Driver Version Tested Against" to 20.5.0:
> 
> 20.5.0 Driver Runtimes has been installed on nimi-bot
> Update applies to NI-DCPower, NI-SWITCH, and NI-DMM, whose latest released Runtime is 20.5.0

*  NI Switch Executive is not NI-SWITCH.
* Reword description to be less redundant.

Reply:
This was a mistake! I should have changed NI-SWITCH.